### PR TITLE
Update quick-start to link to Overview instead of Brokers

### PR DESCRIPTION
**Pull request by pselvana**
The Brokers page was removed from the Console as of a commit Feb 10, 2023: https://github.com/redpanda-data/console/commit/af6ec2e82c80f2253b34452bacba7c23f7cc95ed#diff-1ad9afb9ae78a5b01c49b84934789fc3c52eabed6d36b38cb3406cf7acf673a2L47

This changes fixes the broken link and provides context on what information is available on the Overview page that replaced the Brokers page.
Feel free to change the text description of the Overview page if it doesn't meet the documentation style guide.

### CODE REVIEW

File: docs/get-started/quick-start.mdx [hunk 1]
```diff
@@ -423,9 +423,9 @@ To test external connectivity using your local machine:
 To explore your topic, you can use Redpanda Console, which you deployed in the `docker-compose.yml` file and is running locally on port 8080.
 Redpanda Console is a developer-friendly web UI for managing and debugging your Redpanda cluster and your applications.
 
-1. [Open Redpanda Console](http://127.0.0.1:8080/brokers) in a web browser.
+1. [Open Redpanda Console](http://127.0.0.1:8080/overview) in a web browser.
 
-  All your Redpanda brokers are listed along with their IP addresses and IDs.
+  An overview of the cluster status, cluster health and broker details is displayed.
```
Comment (Feediver1) [inline docs/get-started/quick-start.mdx:428]:
```suggestion
  An overview of the cluster status, cluster health, and broker details is displayed.
```

**Review by Feediver1 — APPROVED** [review]
please add the comma from suggested change

**Comment by pselvana** [comment]
> please add the comma from suggested change

@Feediver1 - update has been made and thanks for the quick review!  All checks have passed.  Please commit when you are ready.

**Comment by pselvana** [comment]
Pinging @Deflaimun since they are a code owner.  Can someone commit please?  This fixes a broken link in the Quick Start documentation so currently it is not a great first experience from a trust perspective.

**Comment by Deflaimun** [comment]
Thanks @pselvana

**Pull request merged**